lib/generators/geoblacklight/templates/catalog_controller.rb

```diff
@@ -111,6 +111,15 @@ class CatalogController < ApplicationController
     config.add_facet_field Settings.FIELDS.GEOREFERENCED, :label => 'Georeferenced', :limit => 3
     config.add_facet_field Settings.FIELDS.SOURCE, :label => 'Collection', :limit => 8, :show => false
 
+    # GEOBLACKLIGHT APPLICATION FACETS
+
+    # Map-Based "Search Here" Feature
+    # item_presenter       - Defines how the facet appears in the GBL UI
+    # filter_query_builder - Defines the query generated for Solr
+    # filter_class         - Defines how to add/remove facet from query
+    # label                - Defines the label used in contstraints container
+    config.add_facet_field Settings.FIELDS.SPATIAL_EXTENT, item_presenter: Geoblacklight::BboxItemPresenter, filter_class: Geoblacklight::BboxFilterField, filter_query_builder: Geoblacklight::BboxFilterQuery, within_boost: Settings.BBOX_WITHIN_BOOST, overlap_boost: Settings.OVERLAP_RATIO_BOOST, overlap_field: Settings.FIELDS.OVERLAP_FIELD, label: 'Bounding Box'
```

Comment:
question: This is a gnarly facet field configuration! Why were `within_boost`, `overlap_boost`, and `overlap_field` added as field configs rather than just referenced in the code? Testing?

Reply:
Sure. Chris added those additional args in his last revision. I imagine he felt articulating all these options in the facet field configuration was the most transparent place to capture these settings. I'm fine with it, even though it feels oversized compared to all the other facet definitions.